doc/source/tune/api_docs/schedulers.rst

```diff
@@ -1,38 +1,173 @@
-.. _schedulers-ref:
+.. _tune-schedulers:
 
 Trial Schedulers (tune.schedulers)
 ==================================
 
-FIFOScheduler
-~~~~~~~~~~~~~
+In Tune, some hyperparameter optimization algorithms are written as "scheduling algorithms". These Trial Schedulers can early terminate bad trials, pause trials, clone trials, and alter hyperparameters of a running trial.
```

Comment:
Is there a recommended trial scheduler? Is there one that people should always use?

Reply:
added